src/compiler/diagnosticMessages.json

```diff
@@ -2352,6 +2352,10 @@
         "category": "Error",
         "code": 4092
     },
+    "Extends clause of exported class '{0}' refers to a type whose name cannot be referenced.": {

```

Comment:
A couple of minutes with @RyanCavanaugh turned up another option we liked even more: 'extends' clause of exported class ...

I'll go with that and update all the other error messages to match.